pixeltable/share/publish.py

```diff
@@ -16,38 +17,46 @@
 # These URLs are abstracted out for now, but will be replaced with actual (hard-coded) URLs once the
 # pixeltable.com URLs are available.
 
-PIXELTABLE_API_URL = 'https://internal-api.pixeltable.com'
+PIXELTABLE_API_URL = 'https://dev-internal-api.pixeltable.com'
```

Comment:
internal-api.pixeltable.com doesn't work today will move it to back to internal-api.pixeltable.com once new changes are deployed to it